generic_config_updater/field_operation_validators.py

```diff
@@ -0,0 +1,21 @@
+from sonic_py_common import device_info
+
+def rdma_config_update_validator():
+    version_info = device_info.get_sonic_version_info()
+    build_version = version_info.get('build_version')
+    asic_type = version_info.get('asic_type')
+
+    if (asic_type != 'mellanox' and asic_type != 'broadcom' and asic_type != 'cisco-8000'):
+        return False
+
+    if len(build_version) >= 6:
```

Comment:
>6 [](http://example.com/codeflow?start=29&length=1)

Magic number. Let's make version comparison more robust and easy to read.

Reply:
I updated the version comparison